packages/react/src/patterns/sub-patterns/CardLink/CardLink.js

```diff
@@ -58,7 +61,11 @@ const CardLink = ({
  */
 function optionalContent(content) {
   return !content ? null : (
-    <p className={`${prefix}--card-link__content`}>{content}</p>
+    <div
+      className={`${prefix}--card-link__content`}
+      dangerouslySetInnerHTML={{
+        __html: markdownToHtml(content, { italic: true, allowHtml: true }),
```

Comment:
the markdown utility is opt out, so you would need to do `bold: false`, and `italic: true` isn't needed here...

Reply:
ah right, done!